### DISCUSSION TITLE
Emergency landing tumbled crash? -- inav 6.1.1

### DISCUSSION THREAD

**Discussion by lida2003** [iNavFlight/inav#9183] [Q&A] [answered]
There are quite a lot of discussions about my crash in [What will happen, when RC & GPS signal lost? -- inav 6.1.1 #9167](https://github.com/iNavFlight/inav/discussions/9167)

And Thanks for @Jetrell @breadoven patience , I think I have got much more info than before.

Due to our discussion about [what will happen in emergency landing when fail throttle set to 1000 with airmode enabled? #9181](https://github.com/iNavFlight/inav/discussions/9181), I think it'll be really not that easy to talk about this crash, as:
1. airmode seems to keep the attitude even if zero throttle
2. old doc says it'll be a supprise when `failsafe_throttle `= 1000  ( what I think it should probably write before airmode is implemented )
3. my crash, the quad is tumbled NOT attitude hold drop

So I decided to do an emergency landing test.

**And It crashed, crashed, crashed!!!**

# Preset:
- `failsafe_throttle`  = 1150
- disable GPS in gps tab
- RC channel trigger failsafe, which set to LAND

PS: detailed steps, please consult: https://blog.csdn.net/lida2003/article/details/131849539
# Result:
Video: https://www.bilibili.com/video/BV1tu411V7JY?p=1
Log:  [blackbox_log_2023-07-21_070855.TXT](https://github.com/iNavFlight/inav/files/12115385/blackbox_log_2023-07-21_070855.TXT)

# 
I'm NOT sure about exact order of the crash. But it seems has something to do with gyro and logic.

1. failsafe switch from althold
4. throttle reset to 0
5. gyro suddenly enlarged from ~1 to ~35
6. gyro further enlarged (out of range?????)
7. throttle adjusted to ~57% ???? try to save the quad???
8. gyro dead?????  (Maybe on the ground as it's only about 1meter high for test.)
9. Now it's a mess

![图片](https://github.com/iNavFlight/inav/assets/1296171/2a942ee5-68c6-4d0b-bd4c-31ab732bce5a)

10. I wanna fly back to where I was, it flipped when take off. And all gyro data is zero.
![图片](https://github.com/iNavFlight/inav/assets/1296171/8cfb9181-17cd-4fb0-97ee-bc67e3b59b59)

Any ideas, please kindly let me know!

**Comment by breadoven** [comment, marked as answer]
This looks like the same issue as https://github.com/iNavFlight/inav/issues/8950.

The Emergency Landing starts at 00:44:235. The same issue as above happens at 00:44:288, Attitude values drop to 0, PIDS go crazy ... although the motor outputs look wrong even before that.

Moving to issues and flagging as a bug.

**Reply by lida2003** [reply]
@breadoven  I have read https://github.com/iNavFlight/inav/issues/8950

It's closed by @fermiums22, but I didn't get the point of "Inav 6.1.0 Landing after RTH work is good". Maybe @fermiums22 was using RTH to land!

What if in my case (gps glitch), oh no~~~

Two issues have a similar data trend(just forget about vibration of my quad):

![图片](https://github.com/iNavFlight/inav/assets/1296171/63923f7e-7870-401b-9d90-ad815705ec8f)
![图片](https://github.com/iNavFlight/inav/assets/1296171/068d9500-0abe-4e94-b59b-03276c59790d)


It was found in the log that the gyro raw data is inconsistent with gyro filtered data in the same situation(https://github.com/iNavFlight/inav/issues/8950  & https://github.com/iNavFlight/inav/issues/9184), which you (@breadoven ) have mentioned data  consistency in https://github.com/iNavFlight/inav/issues/8950


---
further discussion on https://github.com/iNavFlight/inav/issues/9184
